README.md

```diff
@@ -2,10 +2,10 @@
 
 ![Badgeyay](/frontend/public/images/Badgeyay-artwork.png)
 
-[![Gitter](https://img.shields.io/badge/chat-on%20gitter-ff006f.svg?style=flat-square)](https://gitter.im/fossasia/badgeyay)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/1ac554483fac462797ffa5a8b9adf2fa?style=flat-square)](https://www.codacy.com/app/fossasia/badgeyay)
-[![Build Status](https://api.travis-ci.org/fossasia/badgeyay.svg?branch=development&style=flat-square)](https://travis-ci.org/fossasia/badgeyay)
```

Comment:
Standard Badges should not be changes

Reply:
I haven't changed the standard badges, I downloaded them from the URL mentioned and just changed their name, they are exactly same.